[CS2103T-W13-3] StudyBuddy

Sahilgat opened the pull request:
@ckb055 @jaesimin @chanjunren @Sam-limyr

dvrylc commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,25 +297,294 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |have as much information on hand as possible |get all the information I have collected and tagged as part of my revision and ensure that I have all the necessary information at hand.
```
Consider rephrasing the "so that I can..." column to avoid repeated information

JunHongT commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,25 +297,294 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+6.	User inputs new cheatsheet’s module.
+
+7.	StuddyBuddy asks for new cheatsheet’s tags.
```
Possible extension: 
8a User might not want to input tag. 
Use Case End

JunHongT commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,25 +297,294 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+2.	User chooses to list all cheatsheet.
+
+3.	StudyBuddy displays all cheatsheets.
```
Possible Extension:
3a StudyBuddy display empty panel, with message "No cheatsheet to display"
Use case end

dvrylc commented on docs/DeveloperGuide.adoc:
```diff
@@ -279,13 +279,15 @@ Refer to the guide <<DevOps#, here>>.
+*Value proposition*: integrate flashcards and note compilers for CS students better than a typical mouse/GUI driven app
 
 [appendix]
 == User Stories
```
- Format is correct and all three parts are present
- Benefits match the functions
- Good variety of low, medium and high priority stories

JunHongT commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,25 +297,294 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+2.	User chooses to edit cheatsheet.
+
+3.	StudyBuddy asks for cheatsheet ID for cheatsheet to be edited.
+
```
Step 2 and 3 are redundant, user should be able to execute action in a single command.

dvrylc commented on docs/DeveloperGuide.adoc:
```diff
@@ -347,8 +618,10 @@ _{More to be added}_
 .  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  A flashcard cannot have more than 10 tags.
+.  The product is not required to confirm if the user has answered the flashcard correctly. Instead, the product lets the user checks the answer and determine himself/herself if he/she has answered it correctly.
+.  The in-built flashcard library is expected to be updated annually, to follow the curriculum of NUS computing modules.
```
These are design constraints, not NFRs

JunHongT commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,25 +297,294 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+2.	User chooses to view cheatsheet.
+
+3.	StudyBuddy asks for cheatsheet ID for cheatsheet to be viewed.
+
```
Step 2 and 3 are redundant. User should be able to execute action in a single command.

dvrylc commented on docs/DeveloperGuide.adoc:
```diff
@@ -347,8 +618,10 @@ _{More to be added}_
 == Non Functional Requirements
```
Consider adding the following NFRs 🤔 
- Able to have 1000 flashcards without sluggish performance
- Flashcards should work without Internet connectivity

JunHongT commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,25 +297,294 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+3.	OS asks user where screenshot should be saved.
+
+4.	User saves screenshot in designated StudyBuddy image folder with custom name.
+
```
Step 1 -4 should be part of the precondition?

JunHongT commented on docs/DeveloperGuide.adoc:
```diff
@@ -295,25 +297,294 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+4.	User saves screenshot in designated StudyBuddy image folder with custom name.
+
+5.	User opens StudyBuddy application.
+
```
Precondition already states that StudyBuddy is opened

JunHongT reviewed (commented):
Might want to consider changing the steps from multi-steps to single-step method.

jaesimin committed:
edits to PPP

jaesimin committed:
edits to my PPP

jaesimin committed:
Merge pull request #326 from Sam-limyr/master

Provide better user feedback when a Note is created.

jaesimin committed:
Merge pull request #327 from chanjunren/branch-refactor

Refactored instances of AddressBook to StudyBuddyPro in LogicManager/ModelManger/UserPref

jaesimin committed:
Added newline at EOF

chanjunren committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into branch-test-cases

jaesimin committed:
check aboutus

jaesimin committed:
Updated PPP links for all in AboutUs

ckb055 committed:
Merge pull request #328 from jaesimin/master

Edits to my PPP & updated links

jaesimin committed:
testing the links

ckb055 committed:
Fix deletion

Sam-limyr committed:
Update ViewRawNoteCommandTest to account for feedback string changes.

Sam-limyr committed:
Comment out tests which don't work.

ckb055 committed:
update DG, UG

Sam-limyr committed:
Merge pull request #329 from ckb055/refactor-addressbook

Modified delete prompt such that no longer use exceptions.

ckb055 committed:
update PPP

ckb055 committed:
update ppp

Sam-limyr committed:
Create AddNoteCommandParser test cases.

Sam-limyr committed:
Create ViewRawNoteCommand test cases.

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

Sam-limyr committed:
Create DeleteNoteCommandParser test cases.

Sam-limyr committed:
Edit checkstyle errors.

Sahilgat committed:
Merge pull request #332 from Sam-limyr/master

Add tests for multiple classes.

Sahilgat committed:
Add tests for Answer, Question and slight additions to groundwork for RemindCommand tests

Sahilgat committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

# Conflicts:
#	src/test/java/seedu/address/logic/commands/CommandTestUtil.java

Sahilgat committed:
Add more tests for Flashcard models

Sahilgat committed:
Checkstyle fix, commented out unused import to be used later

Sahilgat committed:
Checkstyle fix

ckb055 committed:
Added logging and assertions

ckb055 committed:
Fix travis checkstyle

ckb055 committed:
Fixed a typo

ckb055 committed:
Fix typo in PPP

ckb055 committed:
rename productname on website

Sahilgat committed:
Merge pull request #331 from ckb055/refactor-addressbook

Added some assertions and logging, and updated my PPP.

Sam-limyr committed:
Merge pull request #333 from Sahilgat/master

Add tests for some Flashcard models

ckb055 committed:
Added startup message

ckb055 committed:
fix checkstyle

chanjunren committed:
Merge pull request #334 from ckb055/refactor-addressbook

Added startup message

chanjunren committed:
Implemented basic testing for StartTimeTrialCommand, ViewFlashcardCommand
Refactored instances of addressbook to studybuddypro

chanjunren committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into branch-test-cases

# Conflicts:
#	src/test/java/seedu/address/logic/commands/CommandTestUtil.java
#	src/test/java/seedu/address/logic/commands/flashcard/RemindFlashcardCommandTest.java

chanjunren committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into branch-test-cases

Resolved merge conflicts

# Conflicts:
#	src/test/java/seedu/address/logic/commands/CommandTestUtil.java
#	src/test/java/seedu/address/logic/commands/flashcard/RemindFlashcardCommandTest.java

chanjunren committed:
Fixed checkstyle errors

chanjunren committed:
Fixed checkstyle errors

Sam-limyr committed:
Allow punctuations to be used in Note Titles.

Sam-limyr committed:
Merge pull request #335 from chanjunren/branch-test-cases

Implemented basic testing for ShowFlashcardCommand, StartTimeTrialCommand and ViewFlashcardCommand

Sahilgat committed:
Add more tests to test UniqueFlashcardList to cover DuplicateFlashcardTitleException and DuplicateFlashcardQuestionException

Sam-limyr committed:
Edit delete commands to check for whether CommandHistory is empty.

Sam-limyr committed:
Improve DeleteNoteCommandTest.

Sam-limyr committed:
Add and update ViewNoteCommandTest and ViewRawNoteCommandTest.

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

# Conflicts:
#	src/main/java/seedu/address/logic/commands/cheatsheet/DeleteCheatSheetCommand.java
#	src/test/java/seedu/address/logic/commands/CommandTestUtil.java

Sam-limyr committed:
Refactor NoteFragment constants to NoteFeatureUtil.

Sahilgat committed:
Add ScheduleIncrementTests and StatisticsTests

Sahilgat committed:
Add some tests for Flashcard model [Incomplete]

Sahilgat committed:
Fix checkstyle

Sam-limyr committed:
Merge pull request #337 from Sahilgat/master

Add more Flashcard model tests

Sahilgat committed:
Merge pull request #336 from Sam-limyr/master

Add more Note test cases.

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

chanjunren committed:
Added Command Summary to UG
Fixed link in Help Window
Updated time trial implementation in DG

chanjunren committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into branch-test-cases

Sam-limyr committed:
Merge pull request #338 from chanjunren/branch-test-cases

Updated parts of documentation

Sam-limyr committed:
Add tests for Note parser methods and NoteFragment.

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

Sam-limyr committed:
Edit NoteBuilder resources for NoteFragmentTest.

Sam-limyr committed:
Remove trailing whitespace.

Sahilgat committed:
Refactor codebase to remove several person instances, addressbook instances and begin work on refactoring storage

Sam-limyr committed:
Edit NoteTest and NoteFragmentTest.

Sam-limyr committed:
Add use of assertions.

Sam-limyr committed:
Update NoteFeatureUtil with assertions and finish NoteFeatureUtilTest.

Sam-limyr committed:
Edit checkstyle.

Sam-limyr committed:
Remove unused imports.

Sam-limyr committed:
Edit constants in NoteFeatureUtilTest.

Sam-limyr committed:
Edit ModelStub in AddNoteCommandTest to account for assertions made in AddNoteCommand.

Sam-limyr committed:
Add logger.info() calls to multiple classes.

Sahilgat committed:
Add question bank for flashcards, refactor storage and fix bug so application still starts up with missing or corrupt JSON file

jaesimin committed:
Removed html

jaesimin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

chanjunren committed:
Added Logging and assertion statements in certain parts of code
Attempting to allow timetrial command to be overwritten when other commands are inputted

jaesimin committed:
- Added logger and asserts functions into cheatsheet commands (add, delete, edit, etc) and cheatsheet model
- Trying to solve the PPP link bug, renamed 'jaesimin' to 'jaesiminn' (cant solve the 'jaesimin.adoc' not being reflected as .adoc problem)

Sahilgat committed:
Comment out and delete unused person tests, refactor address book stuff into StudyBuddyPro stuff. ToDo: Fix storage tests based on refactor.

jaesimin committed:
- Removed the example (johndoe) PPP
- Edited my PPP

Sahilgat committed:
Refactor and add tests for storage, checkstyle fixes

chanjunren committed:
Any commands (other than show) will stop the time trial,
Show flashes the answer earlier

chanjunren committed:
Updated time trial segment of user guide

Sahilgat committed:
Comment out broken tests for fixing by teammate, fix storage tests

Sahilgat committed:
Checkstyle fixes

Sahilgat committed:
Checkstyle fixes newline at EOF

jaesimin committed:
Finished my PPP

jaesimin committed:
Merge pull request #342 from chanjunren/branch-test-cases

Fixed time trial bugs and updated UG

jaesimin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

jaesimin committed:
Merge pull request #339 from Sam-limyr/master

Add several more Note Tests.

jaesimin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

jaesimin committed:
Merge pull request #340 from Sahilgat/master

Refactor codebase

jaesimin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

Sahilgat committed:
Merge pull request #341 from jaesimin/master

Logging, asserts and PPP edits

jaesimin committed:
Refactor UG to standardise the format

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

jaesimin committed:
Try again for my PPP link

Sam-limyr committed:
Merge pull request #343 from jaesimin/master

Refactor UG to standardise the format

Sahilgat committed:
Update PPP

chanjunren committed:
Updated PPP

chanjunren committed:
Updated PPP

chanjunren committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into branch-test-cases

# Conflicts:
#	docs/UserGuide.adoc

chanjunren committed:
Implemented basic testing for StartTimeTrialCommandTest, StratTimeTrialCommandParserTest and ViewFlashcardCommandParserTest

chanjunren committed:
Updated PPP
Fixed timetrial invalid arguments message not showing on command result display.

Sahilgat committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

chanjunren committed:
Fixed ViewFlashcardCommandTest

chanjunren committed:
Fixed checkstyle errors

Sam-limyr committed:
Refactor getAddressBook() method in Logic.

Sam-limyr committed:
Create AddNoteSequenceDiagram.

chanjunren committed:
Updated PPP

Sam-limyr committed:
Add new Note-related diagrams for the Developer Guide.

ckb055 committed:
update dg ppp

ckb055 committed:
Update UG

Sam-limyr committed:
Update AddNoteCommandParserTest.

jaesimin committed:
Merge pull request #344 from chanjunren/branch-test-cases

Updated documentation and implemented more test cases

jaesimin committed:
Modified diagrams for my PPP, DG and UG

Sam-limyr committed:
Add three more Note-related diagrams.

jaesimin committed:
Solved the issue for cheatsheet's GUI minimising

jaesimin committed:
- Removed the Address book's UG and DG
- Added in FAQ for the resizing issue of our application

jaesimin committed:
edited PPP

jaesimin committed:
More on AboutUs

jaesimin committed:
Edited sam's ppp to check the link

chanjunren committed:
Merge pull request #346 from jaesimin/master

Updated UG and solve some issues

Sahilgat committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

Sam-limyr committed:
Add and modify more Note-related diagrams for the Developer Guide.

jaesimin committed:
testing out aboutus

jaesimin committed:
Testing out readme.md

jaesimin committed:
test readme agn

jaesimin committed:
Edited build.gradle to test the links

ckb055 committed:
Merge pull request #347 from jaesimin/master

Testing the build.gradle

Sam-limyr committed:
Standardize view of 'Expected output:' in UserGuide.

jaesimin committed:
trying links

Sahilgat committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

jaesimin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

Sam-limyr committed:
Update Notes section in DeveloperGuide with diagrams.

Sahilgat committed:
Merge pull request #348 from jaesimin/master

removed some lines in my PPP

Sahilgat committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

# Conflicts:
#	docs/UserGuide.adoc

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

Sam-limyr committed:
Remove trailing whitespace.

jaesimin committed:
fixed the warnings of gradlew

ckb055 committed:
Merge pull request #350 from jaesimin/master

fixed the warnings of gradlew

jaesimin committed:
Finalized my ppp

Sam-limyr committed:
Merge branch 'master' into master

chanjunren committed:
Merge pull request #349 from Sam-limyr/master

Add diagrams to Notes section in DeveloperGuide.

chanjunren committed:
Added pdf of PPP

ckb055 committed:
Update sequence diagram

ckb055 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into final-kb

ckb055 committed:
Fix typos first

Sahilgat committed:
Merge pull request #352 from ckb055/final-kb

Minor updates to UG, DG, PPP, and sequence diagram

ckb055 committed:
Fix bug

ckb055 committed:
Fix taglist bug

Sam-limyr committed:
Update PPP of Sam-limyr and include links to code.

Sam-limyr committed:
Merge branch 'master' of https://github.com/Sam-limyr/main

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

Sam-limyr committed:
Merge pull request #351 from chanjunren/branch-test-cases

Added pdf of PPP

Sam-limyr committed:
Merge pull request #353 from ckb055/final-kb

Fixed taglist bug

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

Sahilgat committed:
DG and PPP changes

Sahilgat committed:
DG and PPP changes

Sahilgat committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

# Conflicts:
#	docs/DeveloperGuide.adoc

Sahilgat committed:
UG, DG updates

Sahilgat committed:
Merge pull request #354 from Sam-limyr/master

Edit PPP of Sam-limyr.

Sahilgat committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

jaesimin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

jaesimin committed:
Renamed the link in build.gradle

Sahilgat committed:
Checkstyle fixes

jaesimin committed:
Failed in travis

chanjunren committed:
Merge pull request #355 from Sahilgat/master

Updates to my PPP, DG and UG

ckb055 committed:
Fix delete prompt flaw

ckb055 committed:
fix conflicts and checkstyle

chanjunren committed:
Merge pull request #357 from ckb055/final-kb

Fix deletion flaw

jaesimin committed:
resolving conflicts

jaesimin committed:
Changed the url back to main

Sam-limyr committed:
Add sample Note data to SampleDataUtil.

Sam-limyr committed:
Edit UserGuide and DeveloperGuide with minor edits.

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

# Conflicts:
#	src/main/java/seedu/address/model/tag/UniqueTagList.java
#	src/main/java/seedu/address/model/util/SampleDataUtil.java

chanjunren committed:
Merge pull request #356 from jaesimin/master

Minor changes

chanjunren committed:
Merge pull request #358 from Sam-limyr/master

Add sample Note data.

jaesimin committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

jaesimin committed:
Added manual testing for automation

Sam-limyr committed:
Add Manual Tests for Notes in DeveloperGuide.

ckb055 committed:
save work

ckb055 committed:
Cleanup UG

ckb055 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into final-kb

# Conflicts:
#	docs/UserGuide.adoc

Sahilgat committed:
UG changes, PPP changes

chanjunren committed:
Updated UG / DG / PPP
Updated HelpWindow to include a summary of commands

chanjunren committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into branch-test-cases

# Conflicts:
#	docs/DeveloperGuide.adoc
#	docs/UserGuide.adoc
#	docs/team/chanjunren.adoc

chanjunren committed:
Merge pull request #359 from jaesimin/master

Added manual testing

chanjunren committed:
Merge pull request #361 from ckb055/final-kb

Clean up UG

chanjunren committed:
Merge pull request #362 from Sahilgat/master

UG changes, PPP changes

chanjunren committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into branch-test-cases

# Conflicts:
#	docs/UserGuide.adoc

ckb055 committed:
update PPP and UG

ckb055 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into final-kb

Sam-limyr committed:
Add Model Class diagram.

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

# Conflicts:
#	docs/DeveloperGuide.adoc

ckb055 committed:
Update Ui.png

ckb055 committed:
Rename Ui.PNG to Ui.png

ckb055 committed:
Merge pull request #363 from chanjunren/branch-test-cases

Update Documentation and HelpWindow

ckb055 committed:
Fix typo

Sahilgat committed:
Merge pull request #360 from Sam-limyr/master

Add Manual Tests for Notes in DeveloperGuide.

Sahilgat committed:
Merge pull request #364 from ckb055/final-kb

Minor changes to PPP, added Notes paragraph on UG

Sahilgat committed:
Fix Ui.png

ckb055 committed:
Merge pull request #365 from Sahilgat/master

Fix Ui.png

Sahilgat committed:
Final Ui.png changes

chanjunren committed:
Updated UG / PPP

ckb055 committed:
commit first

ckb055 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into final-kb

Sahilgat committed:
UG, DG and PPP changes

Sahilgat committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

ckb055 committed:
Merge pull request #366 from Sahilgat/master

UG, DG and PPP changes

ckb055 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into final-kb

ckb055 committed:
Update CheatSheet portion

ckb055 committed:
fix Ui.png at readme.adoc

ckb055 committed:
Merge pull request #367 from chanjunren/branch-test-cases

Updated UG / PPP

chanjunren committed:
Added logging for FlashcardCommands

chanjunren committed:
Merge pull request #368 from ckb055/final-kb

Update UG for cheatsheet portion, update README to display Ui.png

ckb055 committed:
Fix small typo

jaesimin committed:
Merge pull request #370 from ckb055/final-kb

Fix small typo in png

jaesimin committed:
Merge pull request #369 from chanjunren/branch-test-cases

Added logging for FlashcardCommands

Sam-limyr committed:
Modify Note diagram.

chanjunren committed:
Updated test code link in PPP

Sam-limyr committed:
Add include:: tags for Sam-limyr's PPP.

Sam-limyr committed:
Finalize Sam-limyr's PPP.

Sam-limyr committed:
Merge pull request #371 from chanjunren/branch-test-cases

Updated test code link in PPP

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

ckb055 committed:
added testfiles for view

ckb055 committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main into final-kb

ckb055 committed:
Update UG for notes

jaesimin committed:
Merge pull request #372 from ckb055/final-kb

Added testfiles for ViewCheatSheetCommand and its parser & update UG

Sahilgat committed:
Finalize my PPP

Sahilgat committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

Sam-limyr committed:
Edit User Guide slightly.

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

# Conflicts:
#	docs/UserGuide.adoc

ckb055 committed:
Merge pull request #373 from Sahilgat/master

Finalize my PPP

Sam-limyr committed:
Merge branch 'master' of https://github.com/AY1920S1-CS2103T-W13-3/main

Sam-limyr committed:
Push final version of Sam-limyr's PPP.

Sam-limyr committed:
Update checkstyle.

jaesimin committed:
Merge pull request #374 from Sam-limyr/master

Add Sam-limyr's final PPP.

jaesimin committed:
Edited ug

jaesimin committed:
Changed minor texts

jaesimin committed:
Changed Ui.png to more suitable one

ckb055 committed:
Merge pull request #375 from jaesimin/master

Edited ug

Sam-limyr committed:
Edit Note diagram.

jaesimin committed:
Merge pull request #376 from Sam-limyr/master

Edit Note diagram.

Sam-limyr committed:
Edit use cases.

Sahilgat committed:
Merge pull request #377 from Sam-limyr/master

Edit use cases.

ckb055 committed:
Added newline at EOF of sahilgat.adoc

ckb055 committed:
Fix Ui.png for travis checker

Sahilgat committed:
Merge pull request #378 from ckb055/final-kb

Added newline at EOF of sahilgat.adoc
